Gemfile

```diff
@@ -24,6 +26,7 @@ gem 'slim'
 # Assets
 gem 'webpacker', '~>5.2.0' # Transpile app-like JavaScript
 gem 'sass-rails' # SASS
+gem "font-awesome-rails" # Fontawesome
```

Comment:
can we use npm package instead? 🤔  https://www.npmjs.com/package/font-awesome

Reply:
Fixed in 63c2887e0a31c121f3d22501a2ef07811e5da7cc

Could you please explain more why do we need to use npm package instead of gem?

Reply:
Because that isn't a gem, it's just for client, let npm/webpacker handle that ;) 

For more information, you can read on [this article ](https://www.timdisab.com/installing-bootstrap-4-on-rails-6/)